### ISSUE TITLE
[BUG]: LoRA does not support the training of reward models

### ISSUE THREAD

**Issue by young-chao** [hpcaitech/ColossalAI#2877]
### 🐛 Describe the bug

![image](https://user-images.githubusercontent.com/34190033/220584377-0467d526-d1c4-46a6-b4c8-c56a088bff96.png)

### env
OS：ubuntu 20.04
GPU：4 x A10
python==3.9.0
torch==1.13.1-cu116
colossalai==0.2.5

### command
`python train_reward_model.py --pretrain "bigscience/bloom-560m" --lora_rank 16`

**Comment by young-chao** [comment]
I am very confused why the training encounters errors after setting lora_rank, whether it is the wrong way of my use。

**Comment by young-chao** [comment]
I read your modifications, I don't think these modifications can fix the problem, and I noticed that the latest main version of ColossalAI removed the use of LoRA during rm model training, and now I think ColossalAI does not support LoRA at all, right?

**Comment by young-chao** [comment]
![image](https://user-images.githubusercontent.com/34190033/222966149-4e52a20f-d533-4567-94dd-a578b7941ec2.png)
